english/marshmallow-attack/marsh-attack.md

```diff
@@ -6,7 +6,9 @@ reaches for the lighter you quickly jab the fork into the
 
 white fluffy hand. The marshmallow pulls back its hand and
 
-a giant gaping mouth appears and starts to scream.
```

Comment:
Actually, it looks like you did modify a line - you added "The" to the end of this line. (Casey doesn't merge pull requests that modify a line a because the modification looks the same as deleting the old version of the line and adding a new version, and she just doesn't merge anything that deletes lines.)

I'll be happy to merge this, though! :scream: